.gitignore

```diff
@@ -19,3 +19,6 @@ bower.json
 bower_components
 node_modules
 .rubocop-*
+
+# Ignore webstorm preferences
+.idea/
```

Comment:
Should we add this to the codebase? (Not critical since it won't do any harm I guess)